test/Microsoft.ML.Tests/ScenariosWithDirectInstantiation/TensorflowTests.cs

```diff
@@ -353,6 +353,232 @@ public void TensorFlowTransformMNISTConvTest()
             }
         }
 
+        [Fact]
+        public void TensorFlowTransformMNISTLRTemplateTrainingTest()
+        {
+            var model_location = "mnist_lr_model";
+            using (var env = new ConsoleEnvironment(seed: 1, conc: 1))
+            {
+                var dataPath = GetDataPath("Train-Tiny-28x28.txt");
+                var testDataPath = GetDataPath("MNIST.Test.tiny.txt");
+
+                // Pipeline
+                var loader = TextLoader.ReadFile(env,
+                new TextLoader.Arguments()
+                {
+                    Separator = "tab",
+                    HasHeader = false,
+                    Column = new[]
+                    {
+                        new TextLoader.Column("Label", DataKind.Num,0),
+                        new TextLoader.Column("Placeholder", DataKind.Num,new []{new TextLoader.Range(1, 784) })
+
+                    }
+                }, new MultiFileSource(dataPath));
+
+                IDataView trans = CategoricalTransform.Create(env, loader, "OneHotLabel","Label");
+                trans = NormalizeTransform.CreateMinMaxNormalizer(env, trans, "Features", "Placeholder");
```

Comment:
>CreateMinMaxNormalizer [](start = 43, length = 22)

Why do we add a normalizer here?

Reply:
This is the simple TensorFlow LR model which need normalization otherwise gradients blow out.

---
In reply to: [223087660](https://github.com/dotnet/machinelearning/pull/1063#discussion_r223087660) [](ancestors = 223087660)